styles.css

```diff
@@ -1,6 +1,8 @@
 @import url('https://fonts.googleapis.com/css2?family=Poppins&display=swap');
```

Comment:
### Your Menu
![Screenshot from 2022-09-07 01-56-04](https://user-images.githubusercontent.com/93039359/188737265-1c222244-04c3-4edb-9a7f-6da779603826.png)

### Figma Menu
![figma](https://user-images.githubusercontent.com/61919617/188723706-48628942-7791-4468-8133-a5df0bbb8e79.png)
**[Optional]**
- [ ] Your design is missing the blur. Please add it to follow the Figma template :blush: